src/check/parse/IR.zig

```diff
@@ -1678,42 +1678,97 @@ pub const NodeStore = struct {
 
                     return decl_node;
                 },
-                .expr => |expr_stmt| {
-                    const expr = ir.store.getExpr(expr_stmt.expr);
-                    const expr_node = expr.toSExpr(env, ir);
-                    return expr_node;
+                .expr => |expr| {
+                    return ir.store.getExpr(expr.expr).toSExpr(env, ir);
                 },
                 .import => |import| {
-                    var import_node = sexpr.Expr.init(env.gpa, "import");
+                    var node = sexpr.Expr.init(env.gpa, "import");
 
                     // Module Qualifier e.g. `pf` in `import pf.Stdout`
-                    import_node.appendStringChild(
+                    node.appendStringChild(
                         env.gpa,
                         if (import.qualifier_tok) |tok| ir.resolve(tok) else "",
                     );
 
                     // Module Name e.g. `Stdout` in `import pf.Stdout`
-                    import_node.appendStringChild(
+                    node.appendStringChild(
                         env.gpa,
                         ir.resolve(import.module_name_tok),
                     );
 
                     // Module Alias e.g. `OUT` in `import pf.Stdout as OUT`
-                    import_node.appendStringChild(
+                    node.appendStringChild(
                         env.gpa,
                         if (import.alias_tok) |tok| ir.resolve(tok) else "",
                     );
 
                     // Each exposed identifier e.g. [foo, bar] in `import pf.Stdout exposing [foo, bar]`
                     for (ir.store.tokenSlice(import.exposes)) |tok| {
-                        import_node.appendStringChild(env.gpa, ir.resolve(tok));
+                        node.appendStringChild(env.gpa, ir.resolve(tok));
                     }
```

Comment:
This may deserve a subnode just to group the list of exposed identifiers.

Reply:
Done in ea89f3270d